lib/logstash/outputs/http.rb

```diff
@@ -256,6 +255,8 @@ def send_event(event, attempt)
           :backtrace => e.backtrace)
         end
     end
+
+    request.call # Actually invoke the request in the background
```

Comment:
is it worth describing a bit more why this fixes the issue? I'm ok with putting that explanation in the commit, but here would prevent future errors. if someone decides to "refactor" the order of these things